components/message_queue/src/tests/cases.rs

```diff
@@ -19,6 +19,7 @@ use crate::{
 };
 
 #[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+#[ignore]
```

Comment:
It will be apprecaited if you can add reason for the ignore:
```suggestion
#[ignore = "xxxx"]
```